src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/RolesController.kt

```diff
@@ -164,7 +202,7 @@ data class Role(
 
 @Schema(description = "Update Role Name")
 data class RoleNameAmendment(
-  @Schema(required = true, description = "Role Name", example = "[\"DPS_ADM\"]")
+  @Schema(required = true, description = "Role Name", example = "[Central admin")
```

Comment:
extra `[` at beginning here

Reply:
Simon has kindly offered to fix this in his next PR that is waiting behind mine.